package.json

```diff
@@ -17,6 +17,7 @@
 	"scripts": {
 		"test": "xo && ava && tsd",
 		"prepare": "npm run --silent update-types",
+		"update-browser-globals": "node scripts/get-browser-globals.mjs",
 		"update-builtin-globals": "node scripts/get-builtin-globals.mjs",
```

Comment:
Would be nice with `update-globals` script to that would run both of these.

Reply:
Plan to add it after I add script for `node`.